<a href="https://colab.research.google.com/github/routb68/hdlc/blob/main/HDLC_print.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import json

In [ ]:
with open('/content/drive/MyDrive/sample data HDLC /full_data_after_simple_NER_division.json', 'r') as f:
  data = json.load(f)

In [ ]:
print(type(data))

<class 'dict'>


In [ ]:
for k in data:
  print(k)

mirzapur_district_court_complex


In [ ]:
print(type(data['mirzapur_district_court_complex']))

<class 'dict'>


In [ ]:
for z in data['mirzapur_district_court_complex']:
  print(z)

valid
invalid
stats


In [ ]:
print(type(data['mirzapur_district_court_complex']['valid']))

<class 'dict'>


In [ ]:
for z in data['mirzapur_district_court_complex']['valid']:
  print(z)

Streaming output truncated to the last 5000 lines.
Misc_ Civil Appeal_14_201922-01-2020435.txt
Misc_ Civil Appeal_15_201922-01-2020436.txt
Misc_ Civil Appeal_18_201216-03-202157.txt
Misc_civil_40_201913-01-2021527.txt
Criminal Appeal_43_201722-11-2019342.txt
Civil Appeal_75_201419-12-201943.txt
Civil Appeal_37_201706-08-2019180.txt
Civil Appeal_49_201619-11-2019377.txt
Civil Appeal_37_201527-11-2019314.txt
Civil Appeal_108_201214-05-2019806.txt
Civil Reivision_35_201607-05-2019859.txt
Civil Reivision_37_201607-05-2019861.txt
Civil Reivision_36_201607-05-2019860.txt
Civil Reivision_43_201513-08-2019133.txt
Civil Reivision_33_201514-08-2019114.txt
Civil Reivision_600110_201302-08-2019202.txt
Civil Reivision_22_201621-01-2021460.txt
Civil Reivision_84_201625-02-2021144.txt
Civil Reivision_99_201329-07-2019258.txt
Civil Reivision_31_201628-05-2019702.txt
Civil Reivision_78_201721-01-2021461.txt
Civil Reivision_300072_201420-11-2019376.txt
Civil Reivision_34_201730-09-2019717.txt
Civil Reiv

In [ ]:
for x in  data['mirzapur_district_court_complex']['valid']['Criminal Appeal_43_201722-11-2019342.txt']:
  print(x,end="")

1 न्यायालय अपर सत्र न्यायाधीश, <नाम> कक्ष सं0-5, मीरजापुर | उपस्थित : <नाम> <नाम> यादव-प्रथम, एच जेएस फौजदारी अपील सं0 43 सन्‌ 2017 राज्य <नाम> उत्तर प्रदेश बजरिये डी0जी0सी0 (फौजदारी), मीरजापुर | अपीलकर्ता। बनाम 1. बंगा राजभर 2. नाटे उफ <नाम> 3. छोटक उफ <नाम> पुत्रगण दूधनाथ, निवासी आराजी लाईन सुल्तानपुर, थाना <नाम> मीरजापुर | विपक्षीगण | निर्णय 1. विद्वान अवर न्यायालय, मुख्य न्यायिक मजिस्ट्रेट, मीरजापुर द्वारा अपराध सं0 399/ 2017 अन्तर्गत <नाम> ३23, 324, 504 भा0द0सं0 थाना कोतवाली <नाम> जनपद मीरजापुर में पारित आदेश दिनांकित 19.08.2017 से क्षुध होकर अपीलकर्ता/ राज्य <नाम> उत्तर प्रदेश बजरिये डी0जी0सी0 (फौजदारी), मीरजापुर द्वारा प्रश्‍नगत आपराधिक अपील प्रस्तुत की गयी है। 2. अपीलकर्ता की ओर से यह तर्क किया गया है कि विद्वान अवर न्यायालय द्वारा पारित प्रश्‍नगत आदेश दिनांकित 19.08.2017 द0प्र०सं0 की <नाम> 41 (1) उपखण्ड-ए से ई में <नाम> प्रावधानों के विपरीत होने के <नाम> निरस्त किये जाने <नाम> है। माननीय उच्चतम न्यायालय द्वारा अरनेश <नाम> बनाम बिहार राज्य में प्रतिपादित सिद्धान्त तथा द॑प्र0सं0

1 न्यायालय अपर सत्र न्यायाधीश, <नाम> कक्ष सं0-5, मीरजापुर | उपस्थित : <नाम> <नाम> यादव-प्रथम, एच जेएस फौजदारी अपील सं0 43 सन्‌ 2017 राज्य <नाम> उत्तर प्रदेश बजरिये डी0जी0सी0 (फौजदारी), मीरजापुर | अपीलकर्ता। बनाम 1. बंगा राजभर 2. नाटे उफ <नाम> 3. छोटक उफ <नाम> पुत्रगण दूधनाथ, निवासी आराजी लाईन सुल्तानपुर, थाना <नाम> मीरजापुर | विपक्षीगण | निर्णय 1. विद्वान अवर न्यायालय, मुख्य न्यायिक मजिस्ट्रेट, मीरजापुर द्वारा अपराध सं0 399/ 2017 अन्तर्गत <नाम> ३23, 324, 504 भा0द0सं0 थाना कोतवाली <नाम> जनपद मीरजापुर में पारित आदेश दिनांकित 19.08.2017 से क्षुध होकर अपीलकर्ता/ राज्य <नाम> उत्तर प्रदेश बजरिये डी0जी0सी0 (फौजदारी), मीरजापुर द्वारा प्रश्‍नगत आपराधिक अपील प्रस्तुत की गयी है। 2. अपीलकर्ता की ओर से यह तर्क किया गया है कि विद्वान अवर न्यायालय द्वारा पारित प्रश्‍नगत आदेश दिनांकित 19.08.2017 द0प्र०सं0 की <नाम> 41 (1) उपखण्ड-ए से ई में <नाम> प्रावधानों के विपरीत होने के <नाम> निरस्त किये जाने <नाम> है। माननीय उच्चतम न्यायालय द्वारा अरनेश <नाम> बनाम बिहार राज्य में प्रतिपादित सिद्धान्त तथा द॑प्र0सं0 की <नाम> 41/1) के उपखण्ड ए से ई में <नाम> प्रावधानों के विपरीत प्रश्‍नगत आदेश पारित किया गया है। <नाम> 411) के उपखण्ड ए से ई में <नाम> प्रावधानों के अनुसार अभियुक्तगण बंगा राजभर, नाटे उफ <नाम> व छोटक उफ <नाम> को गिरफ्तार किया गया है। इस सम्बन्ध में केस डायरी के पर्चा सं0-1 में गिरफ्तारी के निम्नलिखित <नाम> अभिलिखित है- ए. यह कि मौके <नाम> अभियुक्तगण का भय एवं <नाम> का माहौल बना हुआ था। 2 इस बात की <नाम> <नाम> थी कि प्रतिशोध की <नाम> से मारपीट करते हुए हत्या कारित <नाम> सकते हैं। बी. यह कि मुकदमा उपरोक्त के अभियुकतगण की गिरफतारी किया <नाम> आवश्यक हो गया था कि गवाहान को अभियुक्तगण मार पीट <नाम> गम्भीर <नाम> से घायल व उ्हे प्रभावित <नाम> सकते थे। सी. यह कि अभियुक्तगण के भाग जाने की पूर्ण <नाम> थी, जिससे न्यायालय में उनकी उपस्थिति सुनिश्चित नहीँ की जा सकती | डी. यह कि अपराध के उचित अन्वेषण के लिये अभियुक्तगण की गिरफूतारी आवश्यक थी। वादी मुकदमा मजरूबा सरोजा <नाम> करिया उफ हरिनाथ के 161 दप्र0सं0 के अन्तर्गत दर्ज किये बयान से भी अपराध की गम्मीरता तथा मौके <नाम> अभियुक्तगण की दबंगई तथा डर के तथ्य स्पष्ट हो रहे हैं। अवर न्यायालय के आदेश दिनांक 19.08.2017 अपराध संख्या 399/17 <नाम> 323, 324, 504 भा0द0सं0 थाना <नाम> के द्वारा पुलिस अधीक्षक मीरजापुर को आदेशित किया गया है कि वह विवेचक उपनिरीक्षक मो0 शाहिद खाँ के विरूद्ध विभागीय कार्यवाही <नाम> न्यायालय को अवगत करायें। अभियोजन द्वारा जानबूझकर र्‍्यायालय की अवमानना नहीं की गयी है। अभियोजन द्वारा की गयी कार्यवाही दप्रशसं0 की <नाम> 41(1) एवं अरनेश <नाम> बनाम बिहार राज्य में दिये गये विधिक प्राविधानों के अन्तर्गत सद्भावक है। विद्वान अवर न्यायालय द्वारा प्रश्‍नगत आदेश दिनांकित 19.08.2017 विधिक भावनाओं के विपरीत पारित किया गया है। तदनुसार आलोच्य आदेश दिनांकित 19.08.2017 को निरस्त किए जाने का अनुरोध किया गया है। 3. विपक्षीगण को निर्गत नोटिस की तामीला के बावजूद वे उपस्थित नहीं आये और <नाम> ही उनकी ओर से कोई आपत्ति प्रस्तुत की गयी है। 4. अपीलकर्ता की ओर से उपस्थित विद्वान सहायक जिला शासकीय अधिवक्ता (फौजदारी) को सुना गया तथा पत्रावली <नाम> उपलब्ध अभिलेखों का अवलोकन किया गया| 5. पत्रावली के अवलोकन से दर्शित होता है कि, मु0अ0सं0 399/17 अन्तर्गत <नाम> 323, 324, 504 भा0द0सं0 थाना कोतवाली <नाम> मीरजापुर के अभियोग में दिनांक 3 17.08.2017 को अभियुक्तगण बंगा राजभर, नाटे उफ <नाम> एवं छोटक उफ <नाम> के विरूद्ध प्रथम सूचना रिपोर्ट पंजीकृत की गयी। विवेचक द्वारा अभियुक्तगण उपरोक्त को दि0 18.08.2017 को समय 15.00 बजे गिरफ्तार किया गया तथा दिनांक 19.08.2017 को विद्वान अवर न्यायालय मुख्य न्यायिक मजिस्ट्रेट, मीरजापुर के समक्ष रिमाण्ड <नाम> प्रस्तुत किया गया| 6. विवेचक द्वारा अभियुक्तगण को <नाम> 323, 324, 504 भा0दं0सं0 के अपराध में गिरफ्तार किया गया। अपराध अन्तर्गत <नाम> 323 भा0द0सं0 एक वर्ष के कारावास, <नाम> 324 भा0द0सं0 तीन वर्ष के कारावास एवं <नाम> 504 भा0दं0सं0 दो वर्ष के कारावास की सजा से दण्डनीय है। इस प्रकार अभियुक्तगण के विरूद्ध आरोपित अपराध सात वर्ष से कम अवधि के कारावास की सजा से दण्डनीय हैं| माननीय उच्चतम न्यायालय द्वारा विधिक दृष्टान्त "अरनेश <नाम> बनाम बिहार राज्य” में यह <नाम> व्यवस्था प्रतिपादित की गयी है कि, जहाँ अभियुक्त द्वारा कारित अपराध 7 वर्ष से कम अवधि के कारावास से दंण्डनीय है, वहाँ उसे केवल अपराध करने के आधार <नाम> गिरफ्तार नहीं किया जायेगा। द॑ण्ड प्रकिया <नाम> की <नाम> 41) के उपखण्ड-ए से ई में <नाम> प्रावधानों के उद्देश्य के लिए हो अभियुक्त को गिरफ्तार किया जायेगा, गिरफूतार करने वाला पुलिस <नाम> गिरफ्तारी के आधारों को लेखबद्ध करेगा। 7, विवेचक द्वारा केस डायरी के पर्चा नं0-1 में अभियुक्तगण की गिरफूतारी का यह आधार <नाम> किया गया कि अभियुक्तगण द्वारा मजरूबान करिया सरोज, हरिनाथ <नाम> को अकारण लाठी, डन्डा व चाकू से मारपीट <नाम> गम्भीर <नाम> से घायल <नाम> <नाम> गया, उनके मारने पीटने से भय एवं आतंक का माहौल बना हुआ है। भविष्य में भी वादी मुकदमा व उसके परिवार के साथ प्रतिषोध की <नाम> से मारपीट करते हुए हत्या जैसी ६ [टना कारित <नाम> सकते हैं। अभियुक्तगण की गिरफतारी नहीं की जाती है तो अभियोग के घायलों को मारपीट <नाम> गम्भर <नाम> से घायल <नाम> सकते हैं, जिससे साक्षीगण भी प्रभावित होंगे। मारपीट की घटना के उपरान्त उनके पलायित हो जाने की पूर्ण <नाम> है, जिससे अभियोग के निस्तारण में व्यवधान उत्पन्न होगा। विवेचक द्वारा केस डायरी में आहत सरोजा <नाम> हरिनाथ <नाम> तथा चक्षुदर्शी <नाम> के <नाम> में शटरू, किशन व <नाम> का 4 बयान <नाम> किया गया है, जिससे स्पष्ट है कि अभियुक्तगण की गिरफ्तारी से पूर्व विवेचक द्वारा घटना के सभी आहत व चक्षुदर्शी साक्षियों का कथन <नाम> किया जा चुका है। ऐसी स्थिति में अभियुक्तगण को गिरफ्तार <नाम> किये जाने <नाम> उनके द्वारा गवाहों को प्रभावित किये जाने का आधार विश्वसनीय नहीं है। विवेचक ने अपनी इस उपधारणा के सम्बन्ध में भी कोइ साक्ष्य संग्रहित नहीं किया है कि कथित घटना के <नाम> भय एवं आतंक का माहौल बना हुआ है और यदि अभियुक्तगण को गिरफतार नहीँ किया गया तो वे वादी व उसके परिवार के साथ हत्या जैसी घटना कारित <नाम> सकते हैं| विवेचक द्वारा एक अन्य आधार यह भी <नाम> किया गया कि मारपीट की घटना के <नाम> अभियुक्तगण के पलायित हो जाने की पूर्ण <नाम> है। घटना दिनांक 17.08.2017 को समय 1310 बजे की है और अभियुक्तगण को दिनांक 18.08.2017 को 15.00 बजे गिरफ्तार किया गया है, अर्थात्‌ अभियुक्तगण को घटना के समय गिरफूतार नहीं किया गया है, बल्कि घटना के <नाम> गिरफ्तार किया गया है। ऐसी स्थिति में विवेचक द्वारा अभियुक्तगण के पलायित हो जाने की <नाम> का आधार मात्र अभियुक्तगण के गिरफ्तारी की औपचारिकता पूर्ण करने के लिए <नाम> गया है। विवेचक द्वारा अभियुक्तगण को गिरफ्तारी से पूर्व <नाम> 41क दप्रशसं0 का कोई सूचनापत्र भी प्रेषित नहीं किया गया है। 8. विद्वान अवर न्यायालय मुख्य न्यायिक मजिस्ट्रेट द्वारा यह अभिमत व्यक्त करते हुए कि, विवेचक द्वारा दप्र0सं0 की <नाम> 41 के प्रावधानों का उल्लंघन करते हुए अभियुक्तगण बंगा राजभर, नाटे उफ <नाम> एवं छोटक उफ <नाम> को गिरफूतार किया गया है। दं0प्र०सं0 के प्राविधान एवं माननीय उच्चतम न्यायालय द्वारा अरनेश <नाम> <नाम> बिहार राज्य में घोषित <नाम> व्यवस्था के <नाम> में अभियुक्तगण बंगा राजभर, नाटे उफ <नाम> एवं छोटक उफ <नाम> को न्यायिक अभिरक्षा में निरूद्ध किया <नाम> न्यायसंगत व <नाम> सम्मत नहीं है, उप निरीक्षक मो0 शाहिद खो द्वारा प्रस्तुत रिमाण्ड रार्थनापत्र निरस्त किया गया तथा अभियुक्तगण को इस निदेश के साथ मुक्त किया गया कि न्यायालय या विवेचक जब भी उन्हें आहूत करेंगे, वे उपस्थित होंगे | 9, विद्वान अवर न्यायालय मुख्य न्यायिक मजिस्ट्रेट द्वारा पारित आलोच्य आदेश में यह निष्कर्ष <नाम> करते हुए कि, माननीय उच्चतम न्यायालय द्वारा अरनेश <नाम> <नाम> बिहार राज्य में यह अवधारित किया गया है कि "यदि विवेचक / पुलिस अधिवकारी द्वारा दप्रसं0 के <नाम> 41 के प्राविधानों का अनुपालन किये <नाम> अभियुक्त को गिरफ्तार किया जाता है तो यह न्यायालय की अवमानना होगी और सम्बन्धित विवेचक / पुलिस <नाम> के विरूद्ध विभागीय कार्यवाही की जायेगी।' इस प्रकार में उप निरीक्षक मो0 शाहिद खां द्वारा माननीय उच्चतम न्यायालय द्वारा घोषित <नाम> व्यवस्था का उल्लंघन करते हुए अभियुक्तगण बंगा राजमर, नाटे उफ <नाम> एवं छोटक उफ <नाम> को गिरफूतार किया गया है। अत: पुलिस अधीक्षक, मीरजापुर को आदेशित किया गया कि वह विवेचक / उप निरीक्षक मो0 शाहिद खाँ के विरूद्ध विभागीय कार्यवाही <नाम> न्यायालय को अवगत करायें। आदेश की <नाम> पुलिस अधीक्षक, मीरजापुर को इस आशय से प्रेषित किये जाने <नाम> निदेशित किया गया कि वह इस आदेश का अनुपालन करते हुए भविष्य में मी माननीय उच्चतम न्यायालय की <नाम> व्यवस्था का अनुपालन किया <नाम> सुनिश्चित करेंगे। विद्वान अवर न्यायालय मुख्य न्यायिक मजिस्ट्रेट द्वारा पारित आदेश द॑0प्र०सं0 के प्राविधान एवं माननीय उच्चतम न्यायालय द्वारा अरनेश <नाम> <नाम> बिहार राज्य में घोषित <नाम> व्यवस्था के अनुरूप है। विद्वान अवर न्यायालय मुख्य न्यायिक मजिस्ट्रेट द्वारा पारित आलोच्य आदेश में कोई त्रुटि अथवा अवैधानिकता नहीँ पायी जाती है। उपरोक्त परिस्थितियों में अपीलकर्ता द्वारा प्रस्तुत अपील स्वीकार किये जाने <नाम> नहीं है। आदेश अपीलकर्ता द्वारा प्रस्तुत अपील निरस्त की जाती है। विद्वान अवर न्यायालय, मुख्य न्यायिक मजिस्ट्रेट, मीरजापुर द्वारा अपराध सं0 399//2017 अन्तर्गत <नाम> 323, 324, 504 मा0द0सं0 थाना कोतवाली <नाम> जनपद मीरजापुर में पारित आदेश दिनांकित 19.08.2017 की <नाम> की जाती है। निर्णय की <नाम> विद्वान अवर न्यायालय, मुख्य न्यायिक मजिस्ट्रेट, मीरजापुर को अग्निम कार्यवाही <नाम> प्रेषित की जाय | दिनांक, 22.11.2019 <नाम> <नाम> यादव-प्रथम) अपर सत्र न्यायाधीश, <नाम> कक्ष सं0-5, मीरजापुर | निर्णय आज खुले न्यायालय में मेरे द्वारा हस्ताक्षरित व दिनांकित होकर उद्घोषित किया गया| दिनांक, 22.11.2019 <नाम> <नाम> यादव-प्रथम) अपर सत्र न्यायाधीश, <नाम> कक्ष सं0-5, मीरजापुर |

In [ ]:
for x in  data['mirzapur_district_court_complex']['valid']['Civil Appeal_37_201706-08-2019180.txt']:
  print(x,end="")

1 न्यायालय अपर जिला जज, <नाम> कक्ष सं0-5, मीरजापुर | उपस्थित : <नाम> <नाम> <नाम> यादव-प्रथम, एच जे.एस. सिविल अपील सं0 37 सन्‌ 2017 <नाम> <नाम> <नाम> उम्र लगभग 22 वर्ष पुत्र <नाम> राजवंश <नाम> निवासी मौजा <नाम> परगना भुईली, तहसील <नाम> जिला मीरजापुर | अपीलार्थी। बनाम 1. राजवंश <नाम> उम्र 53 वर्ष पुत्र स्व0 दयाशंकर <नाम> 2. <नाम> <नाम> <नाम> पत्नी राजवंश <नाम> समस्त निवासीगण मौजा <नाम> परगना भुइली, तहसील <नाम> जिला मीरजापुर | प्रतिवादी /रस्पाण्डेण्ट्स | निर्णय 1. अपीलकर्ता <नाम> <नाम> <नाम> के द्वारा प्रश्‍्नगत सिविल अपील, विद्वान अवर न्यायालय सिविल जज (जू0डि0), <नाम> मीरजापुर, द्वारा मूलवाद सं0 236 सन्‌ 2014 <नाम> <नाम> <नाम> बनाम राजवंश आदि में पारित एक पक्षीय निर्णय व आज्ञप्ि दिनांकित 07.09.2017 से क्षुब्ध होकर योजित की गयी है। 2, सं्षेप में तथ्य निम्नानुसार है- अपीलार्थी/वादी <नाम> <नाम> <नाम> द्वारा विद्वान अधीनस्थ न्यायालय के समक्ष मूलवाद सं0 236 सन्‌ 2014 <नाम> <नाम> <नाम> बनाम राजवंश <नाम> आदि प्रतिवादीगण के विरूद्ध दो किता बैनामा दिनांकित 28.07.2014 व 31.07.2014 बहक <नाम> <नाम> 

1 न्यायालय अपर जिला जज, <नाम> कक्ष सं0-5, मीरजापुर | उपस्थित : <नाम> <नाम> <नाम> यादव-प्रथम, एच जे.एस. सिविल अपील सं0 37 सन्‌ 2017 <नाम> <नाम> <नाम> उम्र लगभग 22 वर्ष पुत्र <नाम> राजवंश <नाम> निवासी मौजा <नाम> परगना भुईली, तहसील <नाम> जिला मीरजापुर | अपीलार्थी। बनाम 1. राजवंश <नाम> उम्र 53 वर्ष पुत्र स्व0 दयाशंकर <नाम> 2. <नाम> <नाम> <नाम> पत्नी राजवंश <नाम> समस्त निवासीगण मौजा <नाम> परगना भुइली, तहसील <नाम> जिला मीरजापुर | प्रतिवादी /रस्पाण्डेण्ट्स | निर्णय 1. अपीलकर्ता <नाम> <नाम> <नाम> के द्वारा प्रश्‍्नगत सिविल अपील, विद्वान अवर न्यायालय सिविल जज (जू0डि0), <नाम> मीरजापुर, द्वारा मूलवाद सं0 236 सन्‌ 2014 <नाम> <नाम> <नाम> बनाम राजवंश आदि में पारित एक पक्षीय निर्णय व आज्ञप्ि दिनांकित 07.09.2017 से क्षुब्ध होकर योजित की गयी है। 2, सं्षेप में तथ्य निम्नानुसार है- अपीलार्थी/वादी <नाम> <नाम> <नाम> द्वारा विद्वान अधीनस्थ न्यायालय के समक्ष मूलवाद सं0 236 सन्‌ 2014 <नाम> <नाम> <नाम> बनाम राजवंश <नाम> आदि प्रतिवादीगण के विरूद्ध दो किता बैनामा दिनांकित 28.07.2014 व 31.07.2014 बहक <नाम> <नाम> को मंसूख किये जाने <नाम> संस्थित किया गया है। 3. वाद-पत्र में वादी के कथनानुसार प्रतिवादी सं0-1 वादी का सगा पिता है तथा प्रतिवादी सं0 2 वादी की मौसी व सौतेली माता है। वादी की माता स्व0 विद्यावती पत्नी राजवंश <नाम> की मृत्यु सन्‌ 1995 में हो गयी थी, उस समय वादी की उम्र लगभग 12 <नाम> थी। वादी का <नाम> दिनांक 22121995 को <नाम> बहादुर <नाम> अस्पताल रामनगर में हुआ था| वादी की माता की मृत्यु के पश्चात्‌ वादी की देखभाल उसके बड़े <नाम> 2 मागवत <नाम> की पली <नाम> <नाम> जिनकी कोई औलाद नहीं थी, वह तथा वादी की सगी बुआ धर्मशीला पत्नी विवेकानन्द <नाम> करने लगीं| वादी के पिता राजवंश ने उसकी माता की मृत्यु के पश्चात्‌ वादी की सगी मौसी से विवाह <नाम> लिया। वादी के पिता तथा पिता के सगे <नाम> <नाम> ने अपनी पैतृक सम्पत्ति कृषिक <नाम> काफी अरसा बेच चुके हैं| वादी की बड़ी दादी <नाम> <नाम> ने दिनांक 25.05.1995 को अपनी समस्त जायदाद का पंजीकृत वसीयतनामा वादी के हक में <नाम> <नाम> तथा दिनांक 23.07.1999 को फौत <नाम> गयीं। इस प्रकार वादी की वसीयत वाली सम्पत्ति <नाम> प्रतिवादीगण से कोई वास्ता व सरोकार नहीं था। प्रतिवादीगण <नाम> तो कमी वादी की परवरिश किये और <नाम> ही कमी कोइ खर्चा देते है। समी खर्च का निर्वहन वादी की बुआ व फूफा ही करते हैं। वादी का नाम स्व0 <नाम> <नाम> के स्थान <नाम> राजस्व अभिलेख में नामांतरण हुआ तथा पिता होने के नाते राजवंश को जरिए संरक्षक दर्ज हुआ। माह <नाम> 2014 को अफवाहन जानकारी हुई कि प्रतिवादी सं0-1 ने वादी की जायदाद का बैनामा प्रतिवादिनी सं0-2 को कृमशः दिनांक 28.07.2014 तथा 31.07.2014 को <नाम> दिया। जब वादी ने बैनामा मंसूख कराने के लिए कहा, तब प्रतिवादी ने0-1 ने इन्कार <नाम> दिया। बैनामा में वादी को नाबालिग तथा उम्र 10 वर्ष लिखा गया है, जो कि <नाम> तरह असत्य है| उक्त बैनामा प्रभावहीन व शून्य है। अत: बैनामा निरस्त किया जाय | 4, विद्वान अवर न्यायालय द्वारा प्रतिवादीगण को अपना जवाबदावा दाखिल <नाम> उत्तर देने <नाम> आहूत किया गया, परन्तु प्रतिवादीगण के अनुपस्थित रहने के <नाम> आदेश दिनांकित 05.11.2015 के द्वारा वाद की कार्यवाही प्रतिवादीगण के विरूद्ध एकपक्षीय <नाम> से अग्रसारित की गयी| 5. अभिलेखीय साक्ष्य के <नाम> में सूची 7ग से दो अदद बैनामा दस्तावेज पक्की नकल दिनांकित 28.07.2014 व 31.07.2014 तथा एक अदद <नाम> प्रमाणपत्र कौ छायाप्रति, सूची 12ग से एक अदद मूल <नाम> प्रमाणपत्र, सूची 13ग से <नाम> अदद नकल खतौनी बावत खाता सं0 184, 183, 121, 182 दाखिल की गयी। सूची 14ग से एक अदद पक्की नकल वसीयतनामा दि0 24.05.99 नविस्ता <नाम> <नाम> विधवा बहक वदस्त <नाम> <नाम> <नाम> पुत्र राजवंश पाण्डे एवं सूची 15ग से एक अदद परिवार रजिस्टर की पक्की नकल व एक 3 अदद छायाप्रति मा0 शि0 परिषद, उ0 प्र0 हाइस्कूल प्रमाणपत्र, 2013 द्वारा <नाम> <नाम> <नाम> दाखिल किया गया है। 8. विद्वान अधीनस्थ न्यायालय के समक्ष वादी की ओर से मौखिक साक्ष्य में पी0डब्लू०-1 के <नाम> में स्वयं का शपथपत्र कागज सं0 10क तथा पी0डब्लू0-2 के <नाम> में <नाम> धर्मशीला का शपथपत्र कागज सं0 11क प्रस्तुत किया गया है। 7. विद्वान अधीनस्थ न्यायालय द्वारा वादी की ओर से दाखिल दस्तावेजी साक्ष्य तथा मौखिक साक्ष्य के आधार <नाम> यह निष्कर्ष निकालते हुए कि बैनामा के समय वादी नाबालिग था और नाबालिग का नेसर्गिक संरक्षक उसके हित के लिए उसके नाम की सम्पत्ति बेच सकता है, पारित निर्णयादेश दिनांकित 07.09.2017 के द्वारा दावा वादी सव्यय खारिज किया गया, जिससे क्षुध होकर अपीलार्थी/वादी द्वारा यह सिविल अपील योजित की गयी है। 8. अपीलार्थ/वादी द्वारा अपील में यह कथन किया गया है कि विद्वान अवर न्यायालय द्वारा प्रश्‍नत निर्णय व आज्ञप्ति पत्रावली <नाम> उपलब्ध तथ्यों एवं साक्ष्यों के सर्वथा विपरीत पारित किया गया है। विद्वान अवर न्यायालय द्वारा पारित एक पक्षीय निर्णय व आज्ञप्ति <नाम> के स्थापित सिद्धान्तों के प्रतिकूल है। प्रश्‍नगत आज्ञप्ति पारित करने में विद्वान अवर न्यायालय द्वारा विधिक मरितष्क का कत्तई प्रयोग नहीं किया गया है। प्रतिवादीगण,/ रेस्पाण्डेन्ट्स अपीलार्थी के पिता व सौतेली माता हैं, वर्ष 199 में अपीलार्थी की पैदाईश के करीब 9-10 <नाम> के <नाम> ही अपीलार्थी की माता का देहान्त हो जाने के <नाम> अपीलार्थी की परवरिश उसकी दादी व बुआ द्वारा की गयी और अपीलार्थी की दादी <नाम> गिरजा <नाम> पत्नी <नाम> <नाम> लावल्द होने की वजह से अपनी जायदाद का पंजीकृत वसीयतनामा अपीलार्थी के पक्ष में तहरीर <नाम> <नाम> था। गिरजा <नाम> की मृत्यु के <नाम> अपीलार्थी अपनी बुआ <नाम> धर्मशीला के साथ उनके घर जिला चन्दौली रहने लगा और अपीलार्था के पिता रेस्पान्डेन्ट सं0-1 ने वसीयतशुदा जायदाद <नाम> अपीलार्थी का नाम अपने को उसका संरक्षक होना दिखाकर कागजात <नाम> में दर्ज <नाम> लिया। कागजात <नाम> में संरक्षक की हेसियत से दर्ज अपने नाम के इन्द्राज का नाजायज फायदा लेते हुये रेस्पान्डेन्ट सं0-1 ने अपीलार्थी को नाबालिग होना जाहिर करते हुये 4 बहैसियत संरक्षक अपीलार्थी की जायदाद का अन्तरण जरिये पंजीकृत विकृय विलेख दिनांक 28.07.2014 व 31.07.2014 बहक रेस्पाण्डेन्ट सं0-2 को नुमाइशी प्रतिफल की अदायगी दिखाकर <नाम> <नाम> गया। अपीलार्थी को प्रश्‍नगत अन्तरण की जानकारी <नाम> 2014 में अपने पैतृक निवास आने <नाम> हुई है। प्रतिवाद ने उसे रद्द व मंसूख कराने का निवेदन किया, लेकिन रेस्पाण्डन्ट्स ने इन्कार <नाम> दिया। विद्वान अवर न्यायालय ने उपरोक्त तथ्यों की अनदेखी करते हुए वाद में निहित सम्पूर्ण तथ्यों एवं परिस्थितियों <नाम> वगैर कोई विचार किये केवल इस अभिमत के साथ अपीलार्थी का वाद एक पक्षीय तौर <नाम> खारिज <नाम> <नाम> कि वादी अपनी उम्र बैनामा की <नाम> <नाम> वयस्क होने का नहीं साबित <नाम> सका है। विद्वान अवर न्यायालय ने अपीलारथी द्वारा प्रस्तुत अभिलेखीय व मौखिक साक्ष्य की पूर्णतया अनदेखी करते हुये संक्षिप्त आदेश के माध्यम से वादी का वाद गलत तरीके से खारिज करके विधिक भूल कारित किया है, जो किसी <नाम> में संधारणीय नहीं है। विद्वान अवर न्यायालय ने वादी/अपीलार्थी द्वारा उल्लिखित किये गये बैनामा मंसूखी के आधारों <नाम> <नाम> तो कोई विचार व्यक्त किया और <नाम> ही उस <नाम> विचार नहीं किये जाने का कोई <नाम> ही उल्लिखित किया है। उपरोक्त परिस्थितियों में विद्वान अधीनस्थ न्यायालय द्वारा पारित निर्णय व आज्ञप्ति दिनांकित 07.09.2017 निरस्त किये जाने <नाम> है। 9, विपक्षीगण को नोटिस प्रेषित किये जाने के बावजूद <नाम> तो वे उपरिथत आये और <नाम> ही उनकी ओर से कोई लिखित आपतित प्रस्तुत की गयी। 10. अपीलार्थी के विद्वान अधिवक्ता की बहस सुनी गई तथा पत्रावली <नाम> उपलब्ध अभिलेखों एवं आलोच्य निर्णयादेश का अवलोकन किया गया। 11. अपीलाथी/वादी की ओर से प्रस्तुत किये गये अभिकथन/ अभिवचन के आधार <नाम> प्रस्तुत अपील के निर्णय <नाम> निम्न अवधारित विन्दु विरचित किया जाता है- 1. क्या वादपत्र में किये गये कथनों के आधार <नाम> बैनामा दिनांकित 28.07.2014 व 31.07.2014 बहक <नाम> <नाम> पत्नी राजवंश <नाम> मंसूख किये जाने <नाम> है? 12. इस सम्ब में वादी/अपीलाथी <नाम> <नाम> <नाम> का कथन है कि प्रतिवादी सं0-1 राजवंश <नाम> वादी का सगा पिता है तथा प्रतिवादी सं0 2 <नाम> <नाम> वादी की 5 मौसी व सौतेली माता है। वादी की माता स्व0 विद्यावती पत्नी राजवंश <नाम> की मृत्यु सन्‌ 1995 में हो गयी थी, उस समय वादी की उम्र लगभग 12 <नाम> थी। वादी का <नाम> दि0 22121995 को <नाम> बहादुर <नाम> अस्पताल रामनगर में हुआ था। वादी की माता की मृत्यु के पश्चात्‌ वादी की देखभाल उसके बड़े <नाम> <नाम> <नाम> की पत्नी <नाम> <नाम> जिनकी कोई औलाद नहीं थी, वह तथा वादी की सगी बुआ धर्मशीला पली विवेकानन्द <नाम> करने लगीं। वादी के पिता राजवंश ने उसकी माता की मृत्यु के पश्चात्‌ वादी की सगी मौसी से विवाह <नाम> लिया। वादी की बड़ी दादी <नाम> <नाम> ने दि0 25.05.1995 को अपनी समस्त जायदाद का पंजीकृत वसीयतनामा वादी के हक में <नाम> <नाम> तथा दि0 23.07.1999 को फौत <नाम> गयीं। इस प्रकार वादी की वसीयत वाली सम्पत्ति <नाम> प्रतिवादीगण से कोई वास्ता व सरोकार नहीं था। वादी का नाम स्व0 <नाम> <नाम> के स्थान <नाम> राजस्व अभिलेख में नामांतरण हुआ तथा पिता होने के नाते राजवंश का नाम जरिये संरक्षक दर्ज हुआ। माह <नाम> 2014 को अफवाहन जानकारी हुई कि प्रतिवादी सं0-1 ने वादी की जायदाद का बैनामा प्रतिवादिनी सं0-2 को कमश: दिनांक 28.07.2014 तथा 31.07.2014 को <नाम> दिया। जब वादी ने बैनामा मंसूख कराने के लिए कहा, तब प्रतिवादी न॑0-1 ने इन्कार <नाम> दिया| बैनामा में वादी को नाबालिग तथा उम्र 10 वर्ष लिखा गया है, जो कि <नाम> तरह असत्य है। उक्त बैनामा प्रभावहीन व शून्य है। 13, प्रतिवादीगण की ओर से वादी द्वारा वादपत्र में किये गये कथनों के विपरीत कोई जवाबदेही दाखिल नहीं की गयी और <नाम> ही वह विद्वान अधीनस्थ न्यायालय व इस न्यायालय के समक्ष उपस्थित हुये | 14. वादी द्वारा वादपत्र में किये गये अपने अभिकथनों के समर्थन में पंजीकृत वसीयतनामा दिनांकित 24.05.1999 की पक्की नकल कागज सं0 14ग मूल पत्रावली में दाखिल की गयी है। वसीयतकत्री गिरजा <नाम> की मृत्यु के उपरान्त राजस्व अभिलेखों में वसीयतशुदा आराजी <नाम> वादी <नाम> <नाम> <नाम> का नाम तथा संरक्षक के <नाम> में पिता वादी राजवंश <नाम> का नाम दर्ज हुआ। पत्रावली के अवलोकन से इस विन्दु <नाम> उभय पक्ष के बीच कोई विवाद दृष्टिगत नहीं होता| दिनांक 28.07.2014 व 31.07.2014 को प्रतिवादी सं0-1 राजवंश <नाम> द्वारा वादी को नाबालिग दशति हुये उसके संरक्षक की 6 हैसियत से वसीयतशुदा आराजी का पंजीकृत बैनामा प्रतिवादी सं0-2 <नाम> <नाम> (वादी की सौतली मॉ) के पक्ष में निष्पादित <नाम> <नाम> गया। वादी का कथन है कि बैनामा निष्पादित किये जाने के <नाम> <नाम> वह वयस्क हो चुका था, लेकिन पुरानी प्रविष्टियों के आधार <नाम> प्रतिवादी सं0-1 ने उसकी <नाम> का बैनामा <नाम> प्रतिफल के प्रतिवादी सं0-2 के पक्ष में निष्पादित <नाम> दिया। वादी का यह भी कथन है कि प्रश्‍नगत बैनामा उसकी <नाम> को हड़पने के लिये साजिशन किया गया है। वादी/अपीलार्थी की ओर से अपनी आयु के सम्बन्ध में <नाम> बहादुर <नाम> चिकित्सालय, रामनगर, वाराणसी द्वारा जारी <नाम> प्रमाणपत्र दिनांकित 10.09.2014 कागज सं0 12ग/2, नकल परिवार रजिस्टर कागज सं0 15/2 , हाई स्कूल प्रमाणपत्र की छायाप्रति कागज सं0 15ग/3 पत्रावली में दाखिल की गयी है। विद्वान अधीनस्थ न्यायालय द्वारा हाइस्कूल प्रमाणपत्र में <नाम> जन्मतिथि 07.08.1998 को विश्वसनीय मानते हुये वादी को बैनामा निष्पादित किये जाने की <नाम> <नाम> अवयस्क होना पाया गया और उक्त आधार <नाम> यह निष्कर्ष निकालते हुये कि संरक्षक को अवयस्क की सम्पत्ति उसके हित में अन्तरित करने का अधिकार है, वादी का वाद खारिज <नाम> <नाम> गया। जहाँ <नाम> बैनामा की <नाम> <नाम> वादी/अपीलार्थी के अवयस्क होने का प्रश्‍न है, नकल परिवार रजिस्टर कागज सं0 15ग/2 में वादी <नाम> की जन्मतिथि दिनांक 13.04.1990 <नाम> है, साथ हो उसकी माँ विद्यावती की मृत्यु दिनांक 31.12.1995 को होना <नाम> है। वादी द्वारा अपने साक्ष्य शपथपत्र में भी इसी तथ्य का कथन किया गया है। इसके अतिरिक्त यह भी उल्लेखनीय है कि वादी/अपीलार्थी के पक्ष में दिनांक 24.05.1999 को पंजीकृत वसीयतनामा कागज सं0 14ग निष्पादित किया गया है, जिसमें वादी के मॉ की मृत्यु लगभग 4 वर्ष पूर्व हो जाने का अंकन है। वादी की ओर से दाखिल <नाम> प्रमाणपत्र कागज सं0 12ग/2 में वादी की जन्मतिथि 2212.95 <नाम> है। उक्त तथ्य का कोई खण्डन प्रतिवादीगण की ओर से न्यायालय के समक्ष उपस्थित होकर नहीं किया गया है। ऐसी रिथिति में जबकि उसकी माँ की मृत्यु दिनांक 31.12.1995 को होना <नाम> है, वादी की जन्मतिथि 07.08.1998 होने का कोई प्रश्‍न नहीं उठता। उल्लेखनीय है कि हाईस्कूल प्रमाणपत्र में वादी <नाम> की जन्मतिथि 07.08.1998 <नाम> है| इस प्रकार पत्रावली में उपलब्ध साक्ष्यों से यह निष्कर्ष निकलता है कि वादी/अपीलार्थी <नाम> <नाम> 7 <नाम> का <नाम> दिनांक 31.12.1995 (उसकी माँ की मृत्यु की <नाम> से पूर्व हुआ था और वादी/अपीलार्थी बैनामें निष्पादित किये जाने के समय वयस्क हो चुका था। यदि थोड़ी देर के लिये वादी के हाईस्कूल प्रमाणपत्र में <नाम> जन्मतिथि 07.08.1998 को पढ़ा जाय तो भी बैनामे निष्पादन के समय उसकी आयु लगभग 16 वर्ष होती है। प्रतिवादी सं0-1 द्व [रा विकृयपत्र दिनांकित 28.07.2014 में वादी की आयु 10 वर्ष दिखायी गयी है, जबकि विकृयपत्र दिनांकित 31.07.2014 में केवल वादी का नाबालिग होना दर्ज है। ऐसा प्रतीत होता है कि प्रतिवादी सं0-1 द्वारा किसी अनुचित <नाम> को प्राप्त करने के उद्देश्य से बैनामें के समय वादी की आयु 10 वर्ष दर्ज करायी गयी है। प्रतिवादी सं0-1 द्वारा बैनामें में विकय का ऐसा कोई <नाम> दर्ज नहीं किया गया है, जिससे यह निष्कर्ष निकाला जा सके कि वादी की <नाम> का विकय प्रतिवादी सं0-1 द्वारा उसके हित के लिए किया गया है। उल्लेखनीय है कि प्रतिवादी सं0-1 द्वारा कथित बैनामें अपने दूसरी पत्नी (वादी की सौतेली माँ) के पक्ष में निष्पादित किये गये हैँ, जिससे यह निष्कर्ष निकलता है कि प्रतिवादी सं0-1 द्वारा उक्त बैनामें दूषित मानसिकता के तहत किये गये हैं। प्रतिवादी सं0-1 राजवंश <नाम> द्वारा बैनामें निष्पादित किये जाने के पूर्व <नाम> तो किसी न्यायालय की अनुमति ली गयी है और <नाम> ही वादी की सहमति प्राप्त की गयी है। उपरोक्त परिस्थितियों में विद्वान अधीनस्थ न्यायालय द्वारा निकाला गया निष्कर्ष विधिसंगत नहीं है और बैनामें दिनांकित 28.07.2014 व 31.07.2014 बहक प्रतिवादिनी सं0-2 <नाम> <नाम> द्वारा प्रतिवादी सं0-1 राजवंश <नाम> मंसूख किये जाने <नाम> हैं। अवधारित विन्दु सं0-1 वादी/अपीलार्थी के पक्ष में सकारात्मक <नाम> से निर्णित किया जाता है। 15. अवधारित विन्दु सं0-1 वादी/ अपीलार्थी के पक्ष में सकारात्मक <नाम> से निस्तारित किया गया है। तदनुसार अपीला्थी/ वादी की अपील स्वीकार किये जाने <नाम> है। आदेश अपीलार्थी/वादी की अपील स्वीकार की जाती है। विद्वान अधीनस्थ न्यायालय सिविल जज (जू0डि0), <नाम> मीरजापुर, द्वारा मूलवाद सं0 236 सन्‌ 2014 <नाम> <नाम> <नाम> बनाम राजवंश आदि में पारित एक पक्षीय निर्णय व आज्ञप्ति दिनांकित 07.09.2017 अपास्त की जाती है। 8 दावा वादी सव्यय आज्ञप्त किया जाता है। तदनुसार प्रश्‍नगत बैनामें दिनांकित 28.07.2014 व 31.07.2014 बहक प्रतिवादिनी सं0-2 <नाम> <नाम> द्वारा प्रतिवादी सं0-1 राजवंश <नाम> मंसूख किये जाते हैं। निर्णय/आदेश की <नाम> उप निबन्धक कार्यालय <नाम> जिला मीरजापुर को अनुपालनार्थ भेजी जाय | निर्णय की <नाम> के साथ मूलवाद की पत्रावली अधीनस्थ न्यायालय को वापस मेजी जाय। दिनांक 06.08.20190 | <नाम> <नाम> यादव-प्रथम) अपर जिला जज, <नाम> कक्ष सं0-5, मीरजापुर | निर्णय आज खुले न्यायालय में मेरे द्वारा हस्ताक्षरित व दिनांकित होकर उद्घोषित किया गया| दिनांक 06.08.20190 | <नाम> <नाम> यादव-प्रथम) अपर जिला जज, <नाम> कक्ष सं0-5, मीरजापुर |

In [ ]:
print(type(data['mirzapur_district_court_complex']['invalid']))

<class 'dict'>


In [ ]:
for z in data['mirzapur_district_court_complex']['invalid']:
  print(z)

Transfer Application-Civil_193_201902-02-2021391.txt
Original Suit_1500958_201312-09-20191889.txt
Bail Application_426_202015-06-2020186.txt
Criminal Misc_ Cases_1052_202023-02-2021193.txt
Criminal Misc_ Cases_1053_202023-02-2021194.txt
Original Suit_1500777_201528-02-20201027.txt
Restoration_200007_201719-12-202019.txt
Criminal Revision_209_201503-11-2020249.txt
Criminal Revision_137_201810-02-2021288.txt
Cri_ Case_2205436_201504-09-20191937.txt
Complaint Cases_778_201512-09-20191892.txt
Original Suit_197_201404-03-20201025.txt
Anticipatory Bail App__110_201910-10-2019672.txt
Anticipatory Bail App__69_201904-09-20191038.txt
Bail Application_190_202027-02-2020134.txt
Anticipatory Bail App__157_201904-12-2019262.txt
Bail Application_349_202112-04-202136.txt
Anticipatory Bail App__164_201901-01-2020673.txt
Anticipatory Bail App__105_201909-10-2019687.txt
Anticipatory Bail App__7_202007-01-2020615.txt
Anticipatory Bail App__135_201917-12-201970.txt
Anticipatory Bail App__137_201926-11-201

In [ ]:
for x in  data['mirzapur_district_court_complex']['invalid']['Original Suit_1500958_201312-09-20191889.txt']:
  print(x,end="")

In the Court of Additional Civil Judge (JD), Court No. 1, Mirzapur

O/S 958 / 2013

Santosh vs Kumari

Parties called.

The file was presented for disposal of Amin report

Learned Advocate appeared on behalf of the both the parties.

Objection regarding Amin report has been filed by the defendant
and is indexed at 72C. Defendant states that the report along with the
map presented by the Amin in its report is false and is not according to
the actual position. The report of Amin regarding the Rakba No. 0.0130
differs from the actaul position as shown in the Bandobshti. Certain
portion of the land belonging to the defendant is shown by the Amin in

its report. Hence the report of Amin should be dismissed.

Plaintiff has filed rebuttal to the objection and is indexed at 71C.
Plaintiff is asserts that the report of Amin is true and the report is
prepared after due investigation of the disputed property. The report
of Amin filed for the Araji No. 237 Rakba 0.130 hectare is true and
the prope

In [ ]:
for x in  data['mirzapur_district_court_complex']['invalid']['Transfer Application-Civil_193_201902-02-2021391.txt']:
  print(x,end="")

T.A. No. 193 of 2019
Abdul Rahman Vs. Shamsher Ali & other

02.02.2021
Case called out. Learned counsel for the applicant is
present. None is present for the opposite parties.

Heard on transfer application 3-A supported with
affidavit 4-C.

This transfer application has been filed for
transferring the Case No. 57 of 2015, Abdul Rahman Vs.
Shamsher Ali, pending in the court of Addl. Civil Judge, (Sr.
Div.), Mirzapur, on the ground that the court is lying vacant.
Hence, it is prayed that case be transferred to the other
running Court.

Report from the court concerned has been received,
according to which the said case is pending in that court and
court is lying vacant.

Notice was sent to the counsel of opposite party which
has been served but no objection has been filed by him.

In the interest of justice transfer application 3-A is
hereby allowed.

Order

Let the Case No. 57 of 2015, Abdul Rahman Vs.
Shamsher Ali, be recalled from the court concerned and be
transferred to the court of

In [ ]:
print(type(data['mirzapur_district_court_complex']['stats']))

<class 'dict'>


In [ ]:
for z in data['mirzapur_district_court_complex']['stats']:
  print(z)

duplicates
valid
invalid
english document


In [ ]:
print(type(data['mirzapur_district_court_complex']['stats']['duplicates']))

<class 'int'>


In [ ]:
print(data['mirzapur_district_court_complex']['stats']['duplicates'])
print(data['mirzapur_district_court_complex']['stats']['valid'])
print(data['mirzapur_district_court_complex']['stats']['invalid'])
print(data['mirzapur_district_court_complex']['stats']['english document'])

570
6940
373
373
